browser/resources/settings/brave_overrides/basic_page.js

```diff
@@ -277,12 +281,15 @@ RegisterPolymerTemplateModifications({
       sectionSync.insertAdjacentElement('afterend', sectionSearch)
       // Insert extensions
       sectionSearch.insertAdjacentElement('afterend', sectionExtensions)
-      // Insert Wallet
-      sectionExtensions.insertAdjacentElement('afterend', sectionWallet)
-      // Insert IPFS
-      sectionWallet.insertAdjacentElement('afterend', sectionIPFS)
+      // Insert Wallet and IPFS
+      if (isBraveWalletAllowed) {
+        sectionExtensions.insertAdjacentElement('afterend', sectionWallet)
+        sectionWallet.insertAdjacentElement('afterend', sectionIPFS)
+      } else {
+        sectionExtensions.insertAdjacentElement('afterend', sectionIPFS)
+      }
       // Insert Tor
```

Comment:
How about removing this invalid comments while you're touching? :)

Reply:
I think the comment is still correct: we are inserting the Tor section, I just changed where it's inserted after (after IPFS vs after Wallet), which actually makes no difference because this section is grouped with Privacy :-|

Reply:
Oops, you're right. I was confused :)